crates/dojo-lang/src/manifest_test_crate/Scarb.toml

```diff
@@ -1,4 +1,35 @@
 [package]
-cairo_version = "1.1.0"
-name = "manifest_test"
+cairo-version = "2.1.1"
```

Comment:
we can create a symbolic link here to the example ecs rather than copy / pasting it all. ideally we have a much simpler test crate though, that makes it easy to grok changes to the manifest

Reply:
added symbolic link.

> we can create a symbolic link here to the example ecs rather than copy / pasting it all. ideally we have a much simpler test crate though, that makes it easy to grok changes to the manifest

the size of manifest file is mostly due to abi stored inside, and abi is mostly big due to it containing alot of types from core cairo as well, so i am not sure if using a simpler test crate would help alot here.